features/issues/components/IssueList/Issue-list.tsx

```diff
@@ -65,7 +76,8 @@ const PageNumber = styled.span`
 export function IssueList() {
   const router = useRouter();
   const page = Number(router.query.page || 1);
-  // const { inputValue } = useFilters();
+  const { width } = useWindowSize();
+  const isMobileScreen = width <= 1023;
```

Comment:
Introducing this magic number `1023` is a bit risky imo. I'd check if there's a way to calculate the px value from the breakpoint in the theme or at least store this next to it. Otherwise it's very likely a developer would miss to adjust this in case the breakpoint changes for some reason

Reply:
Thanks for the suggestion. I'll look into how to refactor to use the current theme's breakpoint.